services/services.go

```diff
@@ -518,6 +546,59 @@ func slotVizUpdater(wg *sync.WaitGroup) {
 	}
 }
 
+func getEthStoreStatisticsData() (*types.EthStoreStatistics, error) {
+
+	var ethStoreDays []types.EthStoreDay
+	err := db.ReaderDb.Select(&ethStoreDays, `
+		SELECT
+			day,
+			apr,
+			effective_balances_sum_wei,
+			total_rewards_wei
+		FROM eth_store_stats
+		WHERE validator = -1
+		ORDER BY DAY ASC`)
+	if err != nil {
+		return nil, fmt.Errorf("error getting eth store stats from db: %v", err)
+	}
+	daysLastIndex := len(ethStoreDays) - 1
+
+	effectiveBalances := [][]float64{}
+	totalRewards := [][]float64{}
+	aprs := [][]float64{}
+	for _, stat := range ethStoreDays {
+		ts := utils.EpochToTime(stat.Day * 225)
+
+		effectiveBalances = append(effectiveBalances, []float64{
+			float64(ts.Unix()) * 1000,
```

Comment:
Minor: `float64(ts.Unix()) * 1000` is done 3 times per loop. You could do it just once and save the result temporarily.

Reply:
Fixed in new commit